timemachine/lib/custom_ops.pyi

```diff
@@ -0,0 +1,150 @@
+from typing import List, Optional
```

Comment:
> It seems like CMAKE will create this file on the fly?

Correct; the original reason to check it in was so it can be used by the mypy check during linting. But since we're now building the extension during linting anyway to verify that it's up to date, seems like we might be able to avoid checking it in. I'll look into this.

Reply:
Another benefit of checking it in is to get typechecking and completions to work when developing the Python code on non-Linux machines. Not sure how common of a use case that is, though.. It seems like most of us are either developing locally on Linux, or running a remote IDE server on a Linux box?

@maxentile @proteneer @jkausrelay 
 do you know if having the stub file locally be useful for your dev setups?